src/components/sidebar.js

```diff
@@ -39,21 +41,15 @@ const createMenu = (menulist) => {
   );
 };
 
-
 const createSubMenu = (grouplist) => {
   var submenu = [];
-  if(grouplist!=null){
-    for(let i=0; i<grouplist.length; i++){
-      const groupUrl = '/groups/' + grouplist[i].replace(" ","-");
-      submenu.push(
-        <Link to={groupUrl} style={{textDecoration: "none"}}>
-          <Row className="sub-menu-button">
-            # {grouplist[i]}
-          </Row>
-        </Link>
-      );
-    }
-}
+  submenu.push(
+    <div>
+      {grouplist.map((name) => {
+        return <Link to={'/groups/' + name} style={{textDecoration: "none"}}><Row className="sub-menu-button">{name}</Row></Link>
```

Comment:
I think we have to work a bit on url name, but rest is fine!
```suggestion
        return <Link to={'/groups/' + name} style={{textDecoration: "none"}}><Row className="sub-menu-button">{name}</Row></Link>
```